javascript/src/seedsource/components/Layers.jsx

```diff
@@ -1,16 +1,18 @@
 import React from 'react'
 import {get} from "io"
 import config from 'seedsource/config'
+import { morph } from 'utils'
 
 
 class Layers extends React.Component {
     constructor(props) {
         super(props)
 
         this.state = {
-            displayResults: true,
-            displayVariables: true,
-            displaySeedZones: false
+            results: {name: "Results", urlIdentifier: "{serviceId}", awayMessage: "Run the tool to view results", display: true},
```

Comment:
This doesn't belong in state (except the `display` part).

http://brewhouse.io/blog/2015/03/24/best-practices-for-component-state-in-reactjs.html

> Store the simplest possible values to describe a component’s state